spec/views/documents/show.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+RSpec.describe "documents/show.html.erb" do
+  before { populate_default_government_bulk_data }
+
+  describe "history mode banner" do
+    it "shows a history mode banner for political content associated with past government" do
+      edition = build(:edition, :political, government: past_government)
+      assign(:edition, edition)
+      render
+
+      title = I18n.t!("documents.show.historical.title",
+                      document_type: edition.document_type.label.downcase)
+      description = I18n.t!("documents.show.historical.description",
+                            government_name: edition.government.title)
+      expect(rendered)
+        .to include(title)
+        .and include(description)
```

Comment:
`and` is cool, didn't realise it was a thing!

Reply:
Apparently it's not cool enough for L18 😆

Reply:
They're now there in their cool glory 😎